int16[10] data

uint8 SENSOR_FRONT_LEFT 		= 0	#0x70
uint8 SENSOR_FRONT_LEFT_CENTER 	= 1 #0x71
uint8 SENSOR_FRONT_CENTER_LEFT 	= 2 #0x72
uint8 SENSOR_FRONT_CENTER_RIGHT = 3 #0x73
uint8 SENSOR_FRONT_RIGHT_CENTER = 4 #0x74
uint8 SENSOR_FRONT_RIGHT 		= 5 #0x75
uint8 SENSOR_RIGHT 				= 6 #0x76
uint8 SENSOR_REAR_RIGHT 		= 7 #0x77
uint8 SENSOR_REAR_LEFT 			= 8 #0x78
uint8 SENSOR_LEFT 				= 9 #0x79
uint8 SENSOR_COUNT 				= 10

uint8 ULTRASONE_MAX_RANGE		= 250
